documentation/user-interface/fixture-properties-dialog.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+title: Fixture Properties Dialog
+permalink: /documentation/user-interface/fixture-properties-dialog.html
+---
+# Fixture Properties Dialog
+
+The **Fixture Properties** dialog opens when you [create a new fixture](../guides/record-tests/README.md#create-fixtures) and allows you to specify its properties.
+
+The dialog also allows you to [modify](fixture-editor.md#modify-the-fixtures-properties) an existing fixture's properties.
```

Comment:
Maybe rephrase a bit:

```
The **Fixture Properties** dialog opens when you [create a new fixture](../guides/record-tests/README.md#create-fixtures) or [modify an existing one](fixture-editor.md#modify-the-fixtures-properties). Use it to specify or edit fixture's properties.
```

Reply:
I thinks this phrase isn't exactly correct: "... dialog opens when you modify an existing one" because:
1. we open dialog TO MODIFY properties, not WHEN MODIFY
2. we don't modify the fixture, but ITS PROPERTIES.

Reply:
Okay, I just think that two paragraphs is too much for this information, that's why I tried to squeeze it in two shorter sentences.